apps/codeimage/src/core/configuration.ts

```diff
@@ -7,7 +7,13 @@ export const [appEnvironment] = createConfiguration({
   locales: SUPPORTED_LOCALES,
   themes: [],
   languages: [],
-  editorPadding: [16, 32, 64, 128],
+  editorPadding: [
+    {label: '0', value: "0"},
+    {label: '16', value: "16"},
+    {label: '32', value: "32"},
+    {label: '64', value: "64"},
+    {label: '128', value: "128"},
+  ],
```

Comment:
@riccardoperra I kept string because `paddingOptions.controlled` in Select doesn't accept `0` as value because of this line https://github.com/riccardoperra/codeui/blob/main/packages/kit/src/components/Select/createSelectValue.ts#L85

Here I am attaching video of how it behaves when I pass 0 to Select.


https://github.com/riccardoperra/codeimage/assets/6604146/9286397d-351e-4b45-97ca-3873ab955ed1

Reply:
You’re right. I’ll push in the next days a fix in the ui kit about this but at the moment the code you write for this is ok